package.json

```diff
@@ -7,7 +7,8 @@
   "repository": "Developmint/wait-for-netlify-preview",
   "scripts": {
     "lint": "eslint index.js __test__",
-    "test": "yarn run lint && jest --detectOpenHandles",
+    "pretest": "yarn run lint",
+    "test": "jest --detectOpenHandles",
```

Comment:
Longer term--if `main.js` is extracted out into a utility, this would be quite a bit easier to test and this option wouldn't be required.

Reply:
If you want, you can do this in another PR :+1: